tasks/test_add_picklist_values.py

```diff
@@ -0,0 +1,412 @@
+import unittest
+from tasks import add_picklist_values
+from unittest import mock
+import responses
+from cumulusci.salesforce_api.tests.metadata_test_strings import deploy_result
+from cumulusci.tasks.salesforce.tests.util import create_task
+
+# to-do: add otherlast test? assert sorted?
```

Comment:
Per conversation below, the tech debt WI I create for improving the assertions should also consider adding these tests as well